client/components/domains/domain-search-results/style.scss

```diff
@@ -56,9 +56,11 @@
 
 .domain-search-results__transfer-card-link {
 	flex-shrink: 0;
+	flex-grow: 1;
```

Comment:
Could probably use `flex: 1 0;` here. (I think that the shorthand use of `flex` is preferred.)

Reply:
😭maybe for CSS people. I have no idea what it's doing that way. 😛